script/uplift.py

```diff
@@ -425,8 +440,13 @@ def submit_pr(channel, top_level_base, remote_base, local_branch):
             pr_body = 'TODO: fill me in\n(created using `npm run pr`)'
         else:
             pr_title += ' (uplift to ' + remote_base + ')'
-            pr_body = 'Uplift of #' + str(config.master_pr_number) + '\n\n'
-            pr_body += 'Approved, please ensure that before merging: \n'
+            pr_body = 'Uplift of #' + str(config.master_pr_number) + '\n'
+
+            if len(issues_fixed) > 0:
+                for fixed in issues_fixed:
+                    pr_body += (fixed[0] + '\n')
```

Comment:
Since match is made like (resolves|fixes|etc) issue (number), `fixed` has a tuple with 3 items; full string match, text matched for 1st block, text matched for 2nd block